templates/usr/local/lib/owncloud-enforce-permissions.j2

```diff
@@ -5,11 +5,10 @@
 
 # {{ ansible_managed }}
 
-[[ "$UID" != 0 ]] && { echo "You must be root"; exit 1; }
-
```

Comment:
When it was already present in the template, why not. But this is just a detail. You can decided as I also have no check for root in my PR.